src/moveit_visual_tools.cpp

```diff
@@ -343,12 +347,13 @@ void MoveItVisualTools::loadTrajectoryPub(const std::string& display_planned_pat
     return;
 
   // Trajectory paths
-  pub_display_path_ = nh_.advertise<moveit_msgs::DisplayTrajectory>(display_planned_path_topic, 10, false);
-  ROS_DEBUG_STREAM_NAMED(LOGNAME, "Publishing MoveIt trajectory on topic " << pub_display_path_.getTopic());
+  pub_display_path_ = node_->create_publisher<moveit_msgs::msg::DisplayTrajectory>(display_planned_path_topic, 10);
+  RCLCPP_DEBUG_STREAM(LOGGER, "Publishing MoveIt trajectory on topic " << pub_display_path_->get_topic_name());
 
   // Wait for topic to be ready
   if (blocking)
-    waitForSubscriber(pub_display_path_);
+    waitForSubscriber(pub_display_path_, 5.0);
```

Comment:
This waits for 5 seconds, right? Please use inline comments when adding magic numbers, maybe like this:
```suggestion
    waitForSubscriber(pub_display_path_, 5.0 /* seconds */);
```
But should we even change the original behavior without proper error handling?

Reply:
I'll add the comments. I didn't get what you mean by the original behavior?